lib/elixir_auth_github.ex

```diff
@@ -26,17 +34,17 @@ defmodule ElixirAuthGithub do
   appended to the URL.
   """
   def login_url(%{scopes: scopes, state: state}) do
-    login_url()
-    <> "&scope=#{Enum.join(scopes, "%20")}"
-    <> "&state=#{state}"
+    login_url() <>
```

Comment:
these whitespace changes were created by `mix format`. 🙄 
(_it's good to keep our code consistent across projects_)